src/cmd/service.go

```diff
@@ -284,3 +286,28 @@ func NullableString(value *string) *opslevel.Nullable[string] {
 	}
 	return opslevel.NewNullableFrom(*value)
 }
+
+func getService(identifier string) (*opslevel.Service, error) {
```

Comment:
I swear we had something like this in opslevel-go.  If not we should add this to opslevel-go and/or modify `GetService` to work like this.

Reply:
There is no catch all like this in opslevel-go. Something like `GetServiceWithIdentifier(identifier string)` then mark the others as deprecated?

Reply:
I think we can modify `GetService` to accept a string - ID is a type alias for string IIRC - and then have it work if the string passed in is an ID or Alias.  Would need to test a before and after but i think thats a safe non-breaking interface change to the `GetService` method.

Reply:
I tried a few things and could not get this to work